pkg/orchestrator/orchestrator_test.go

```diff
@@ -125,7 +148,7 @@ func TestPipelineSimple(t *testing.T) {
 	t.Log("successfully stopped pipeline")
 
 	// make sure destination file matches source file
-	want, err := os.ReadFile(sourcePath)
+	want, err := os.ReadFile(wantPath)
```

Comment:
The file destination now produces the whole record instead of only the payload so the test expectation had to be adjusted.